CURRENT_SUPPORTED_VERSIONS

```diff
@@ -1,3 +1,4 @@
+2.11
 2.10
 2.9
 2.8
```

Comment:
Keeping 2.8 around is right, but doesn't really _feel_ right 😕

Reply:
Yeah, the script removed it I believe but it needs to stay a little while longer.